AWS High Availability Architecture/README.md

```diff
@@ -0,0 +1,231 @@
+
+To install `httpd` use command:`yum install httpd`
+
+After httpd installation, we need to check whether service is started or not.
+
+To check the httpd service use command:`systemctl status httpd`
+
+> Refer to `Images/Activity 2/Activity2a.png`
+
+And to start the service use the command `systemctl start httpd`
+
+> Refer to `Images/Activity 2/Activity2b.png`
+
+**Activity 2 Task:**
+
+Above command to start the httpd service is temporary as soon as system shutdown service also stops so, try to find the command that will start the httpd service permanent.
+
+### Activity 3: Create an EBS Volume
+
+To Create an EBS Volume we require:
+
+- Availability Zone name
+- Size of Volume that we want to create.
+
+> Syntax:`aws ec2 create-volume --availability-zone ap-south-1a --size 1`
+
+Here `ap-south-1a` is availability zone, you have to give availability zone name where you have launched instance and `1` is the size of volume i.e 1Gb, you can give desired size to your volume.
+
+> Refer to `Images/Activity 3/Activity3a.png` 
+
+### Activity 4: Attach EBS Volume to Instance
+
+To attach the EBS Volume to an Instance we require:
+
+- Volume-id (The volume that we have created in `Activity 3`)
+- instance-id (The instance that we have launched in `Activity 2`)
+- device (Example: /dev/sdf)
+
+> Syntax: aws ec2 attach-volume --volume-id (volume_id) --instance-id (instance_id) --device(ex: /dev/sdf)
+
+> Refer to `Images/Activity 4/Activity4a.png` 
+
+**Activity 4 Task:**
+
+Find volume-id, instance-id using AWS CLI(**Hint**take help of `aws ec2 describe` command)
+
+If not able to find using AWS CLI, you can also get it from AWS WebUi.
+
+### Activity 5: Create and Format a Partition
+
+Now open the instance that you have launched in `Activity 1`
+
+run the command `fdisk /dev/xvdf` to create partition.
+
+After running this commmand press `n` to create new partition and then press `p` and then enter.
+
+To save the partition press `w`.
+
+> Refer to `Images/Activity 5/Activity5a.png` 
+
+Now we need to format the partion.
+
+To format the partition use the command `mkfs.ext4 device_name`
+
+> Refer to `Images/Activity 5/Activity5b.png` 
+
+### Activity 6: Mount /var/www/html directory to partition
+
+We need to mount `/var/www/html` directory to partition because we will create our html page in this directory and it will be safe in this partion even in the time of instance failure.
+
+To mount the directory use the command `mount /dev/xvdf  /var/www/html`
+
+And to check it is mounted or not use command `df -h`
+
+> Refer to `Images/Activity 6/Activity6a.png`  
+
+Now create HTML page, go inside directory `/var/www/html` in your linux instance.
+
+Create a HTML file.
+
+> Syntax:
+```
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+img {
+  width: 100%;
+}
+</style>
+</head>
+<body>
+<img src="html5.png"  width="128" height="128">
+<img src="html5.png" width="128" height="128">
+</body>
+</html> 
+```

```

Comment:
Just add here that: "Create an HTML file with your own content or if you can't think of content, use this one as below"